src/backends/amber/AmberFetcher.php

```diff
@@ -12,7 +12,7 @@ function __construct(iAmberStorage $storage, array $options) {
     $this->storage = $storage;
     $this->assetHelper = new AmberAssetHelper($storage);
     $this->maxFileSize = isset($options['amber_max_file']) ? $options['amber_max_file'] : 1000;
-    $this->headerText = isset($options['header_text']) ? $options['header_text'] : "You are viewing an archive of <a style='font-weight:bold !important; color:white !important' href='{{url}}'>{{url}}</a> created on {{date}}";
+    $this->headerText = isset($options['header_text']) ? $options['header_text'] : "You are viewing a snapshot of <a style='font-weight:bold !important; color:white !important' href='{{url}}'>{{url}}</a> created on {{date}}";
```

Comment:
Snapshot is better terminology + The commit with word snapshot was more recent than archive. Hence, it has been picked.